site/content/contribute/server/developer-setup/ubuntu.md

```diff
@@ -32,7 +32,7 @@
     sudo chmod +x /usr/local/bin/docker-compose
     ```
 
-3. Install Go (modify installation to latest Go version from https://golang.org/dl/):
+3. Install Go (modify installation to latest Go version from <https://golang.org/dl/>):
```

Comment:
What hugo version are you using? I tried your patch with Hugo v0.59.0. The output was the following HTML:
```html
<a href="https://golang.org/dl/):">https://golang.org/dl/):</a>
```

Maybe the link rendering was also changed with v0.60.0?

Reply:
Ah, that makes sense -- @mitchellroe helped identify the v0.60.0 regression, and this likely occurred in that context. @hanzei, do we want to plan an upgrade to v0.60.0 soon, or just shelve that plan for now given ongoing discussion about the myriad sets of documentation tooling we have?

Reply:
I'm not sure what version I tested it under. I'll try it in hugo < 0.60.0 and in hugo >= 0.60.0 and see what happens.

Reply:
You fix works fine with Hugo >= 0.60 :+1: